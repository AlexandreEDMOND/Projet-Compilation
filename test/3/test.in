int main()
{
  int a = 8;
  int b;
  int i;

  while (b == 0)
  {
    if (a > i) 
    {
      b = 1;
    }
    else
    {
      b = 0;
    }

    i++;

  }

  return 0;
}