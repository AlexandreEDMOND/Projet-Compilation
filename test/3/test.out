5
1569
-2
